pkg/operator/apis/monitoring/v1alpha1/types_logs.go

```diff
@@ -412,9 +421,9 @@ type MetricsStageSpec struct {
 	// to a positive float.
 	Action string `json:"action,omitempty"`
 
-	// Buckets to create.
+	// Buckets to create. Bucket values must be convertible to floats.
 	// Only valid for type: histogram.
-	Buckets []int `json:"bucket,omitempty"`
+	Buckets []string `json:"bucket,omitempty"`
```

Comment:
Buckets are supposed to be floats, but Kubernetes doesn't support floats, so....strings it is.

Reply:
Wut? That's intriguing.

Reply:
Does this matter if its float 32/64?

Reply:
Nope, no floats period: https://github.com/kubernetes-sigs/controller-tools/blob/abcf7d8d54b0cca0f42c470b705fd3cfe768ede3/pkg/crd/schema.go#L387

Reply:
Oh, I'm not sure if we need to specify float32/float64 (if your buckets are that small you're probably going to have other problems), but I'm pretty sure these will get read as float64s.

Reply:
Looked into a it a little more and go-jsonnet stores everything as a float64. I'm not super confident in how accurate the string -> float64 conversion is going to be, so I'll add some kind of warning here about risk of precision loss at very large or very small numbers.